docs/tutorial/tutorial.md

```diff
@@ -0,0 +1,126 @@
+# Tutorial
```

Comment:
Can you rename this doc to "getting started" and the folder to "tutorials"?

Reply:
Renamed for consistency. Thank you for the suggestion!